src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/XplatEventLogger.cs

```diff
@@ -199,6 +201,119 @@ private static void LogOnEventWritten(EventWrittenEventArgs eventData)
 
             LogEventSource(eventData.EventId, eventData.EventName, eventData.EventSource.Name, payload);
         }
+#endif //FEATURE_EVENTSOURCE_XPLAT
+
+        [DllImport(RuntimeHelpers.QCall)]
```

Comment:
Can we move all these QCalls definitions to the NativeRuntimeEventSource? They no longer have any clear relationship to XPlatEventLogger.